boards/FSM/Inc/App/configs/App_AcceleratorPedalThresholds.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
+
+#define PAPPS_MAX_PRESSED_VALUE 900U
```

Comment:
Using estimated values for now. The real values should be calibrated to the pedal box.

Reply:
Can we leave a TODO and file a ticket?

Reply:
done 🌮